Servlets/src/main/java/bsu_InstaLife/Servlets/StaticPageServlet.java

```diff
@@ -0,0 +1,31 @@
+package bsu_InstaLife.Servlets;
+
+import javax.servlet.RequestDispatcher;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+@WebServlet("/page")
+public class StaticPageServlet extends HttpServlet {
+    @Override
+    protected void doGet(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
```

Comment:
You declare doGet throws ServletException, IOException
But no one of them can be thrown as you catch it by yourself.
Exception handling is good in the most of the cases.
But what regarding servlets - we need to keep in mind that exception can be handled by servlet container.
So if servlet crashed during execution - all servlet container let us know that something **500**th happened. Furthermore we can even define our custom handlers, that will log it and nice payload to the answer, so if user somehow gets response directly he will see our nice error page from response body instead of boring default error page.